test/apps/uct_info/Makefile.in

```diff
@@ -9,21 +9,17 @@ all: uct_info uct_info_static
 clean:
 	rm -rf uct_info uct_info_static
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-PKG_CONFIG_CMD = PKG_CONFIG_PATH=@libdir@/pkgconfig:$(PKG_CONFIG_PATH) pkg-config
+UCX_CFLAGS = $(CFLAGS) $(shell pkg-config ucx-uct --cflags)
 
-UCX_CFLAGS = $(CFLAGS) $(shell $(PKG_CONFIG_CMD) ucx-uct --cflags)
+UCX_LDFLAGS = $(LDFLAGS) $(shell pkg-config ucx-uct --libs)
 
-UCX_LDFLAGS = $(LDFLAGS) $(shell $(PKG_CONFIG_CMD) ucx-uct --libs)
-
-UCX_STATIC_LDFLAGS = -static $(shell $(PKG_CONFIG_CMD) --libs --static @uct_static_modules@ ucx-uct) -pthread -lrt -ldl
+UCX_STATIC_LDFLAGS = -static $(shell pkg-config --libs --static $(EXTRA_MODULES) ucx-uct) -pthread -lrt -ldl
```

Comment:
why need `-pthread -lrt -ld`?
I would expect UCX's pkg-config files to add such deps

Reply:
> why need `-pthread -lrt -ld`?
> I would expect UCX's pkg-config files to add such deps

autoconf/libtool is covered this, but it can't be used because this makefile is not part of UCX build process

Reply:
It should be part of libucs pkg-config installed file, maybe need to add to ucs's configure.m4

Reply:
> maybe need to add to ucs's configure.m4

no needed: -lrt required for she_open/close API, -ldl us used for symbol lookup, -pthread is needed for -lrt.
libs are added into ucs/ucm.pc files

Reply:
don't we use pthread_xx functions from ucs as well?

Reply:
as I can see no pthread calls from ucs, but librt requires pthread link

Reply:
I'm pretty sure we use pthread locks from ucs rcache, for example

Reply:
yep, sorry... vscode added more filters into search.
yes, a lot of locations where pthread is used

Reply:
added libdl into ucs dependency